addons/account/models/account_journal_dashboard.py

```diff
@@ -215,6 +215,9 @@ def get_journal_dashboard_datas(self):
             (number_waiting, sum_waiting) = self._count_results_and_sum_amounts(query_results_to_pay, currency, curr_cache=curr_cache)
             (number_draft, sum_draft) = self._count_results_and_sum_amounts(query_results_drafts, currency, curr_cache=curr_cache)
             (number_late, sum_late) = self._count_results_and_sum_amounts(late_query_results, currency, curr_cache=curr_cache)
+            to_check_ids = self.env['account.move'].search([('journal_id', '=', self.id), ('to_check', '=', True)])
+            number_to_check = len(to_check_ids)
+            to_check_balance = sum([r.amount for r in to_check_ids])
```

Comment:
use search_read() instead, as the dashboard may compute a lot of information it is important to be aware of potential performances issues

Reply:
I also tried to do it for the bank journals, but its trickier.